.github/workflows/autoformat.yml

```diff
@@ -12,15 +12,13 @@ on:
   push:
     branches-ignore:    
       - 'master'
-
-
+    
 jobs:
 
-  code-and-docs-updates:
+  run:
     runs-on: ubuntu-latest
-    timeout-minutes: 45
     steps:
-
+      - run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
```

Comment:
How could it work before?

Reply:
I guess it didn't :sweat_smile: The lack of this env var is not going to crash the workflow, is simply going to make it miss the cache (see line 35). So I think we simply just never noticed.